src/core/cheats.cpp

```diff
@@ -15,6 +15,7 @@
 #include <iomanip>
 #include <sstream>
 Log_SetChannel(Cheats);
+static std::array<u32, 256> temp_variable; //Used for D7 ,51 & 52 cheat types
```

Comment:
Do we want to make this storage per-code? Or share it between all codes.

Could be worth allocating it dynamically on demand if so.